app/(tabs)/_layout.tsx

```diff
@@ -0,0 +1,69 @@
+import { Tabs } from 'expo-router';
+import React from 'react';
+import { TabBarIcon } from '@/components/navigation/TabBarIcon';
+import { StyleSheet } from 'react-native';
+
+const HIGHLIGHT_COLOR = '#FFCC00';
+const BACKGROUND_COLOR = '#242427';
+const INACTIVE_COLOR = '#777777';
```

Comment:
We can add these to the Colors.ts in constants if we think we'll re-use them in other parts of the app.